Source/Scene/ModelUtility.js

```diff
@@ -491,15 +493,20 @@ define([
         'KHR_techniques_webgl' : true,
         'KHR_materials_unlit' : true,
         'KHR_materials_pbrSpecularGlossiness' : true,
-        'WEB3D_quantized_attributes' : true
+        'WEB3D_quantized_attributes' : true,
+        'EXT_texture_webp' : true
     };
 
-    ModelUtility.checkSupportedExtensions = function(extensionsRequired) {
+    ModelUtility.checkSupportedExtensions = function(extensionsRequired, browserSupportsWebp) {
```

Comment:
Kind of weird to pass `browserSupportsWebp` into here instead of making `ModelUtility.checkSupportedExtensions` return a promise.  `checkSupportedExtensions` is only used in two places (both of which are already async loading of models).

Reply:
We spoke offline, and determined it was simplest to check for WebP support once per Model, cache it, and otherwise wait in the `update` function until that resolves.